### PR TITLE
#419 Upload Profile Picture Button Styles

### PR DESCRIPTION
**Pull request by brittanygay**
Connect #418 
Connect #419 
Requires #408

- Stubbed out and styled the Add Profile Picture button on the SignUp screen
- Fixed a few alignment bugs that I found on the Login/Signup screen

**Note**:
The Sign Up screen still needs to be broken out into separate parts (see video in ticket). I went ahead and added the button to the current screen but once we break these out in #408 we will need to move over the second two inputs (Email and Password), and keep the Add Profile button with the First and Last name inputs. 

- Assigning to @blazebarsamian to review (but don't merge)
- Whichever dev picks up #408 can either hop on my branch or branch off of it to continue working on

screen shot 2019-01-03 at 4 45 55 pm

### CODE REVIEW

**Comment by brittanygay** [comment]
@jwietelmann I just pushed up style updates for the profile picture onboarding flow (styled based on mocks in Abstract here: https://app.goabstract.com/projects/b52ae8a0-8a13-11e8-a66c-3fc17367628a/branches/master/commits/8a0f0edef2897e956ec3ddedb338f65d19310e23/files/AE300282-67A5-42F5-A47D-180F88B20490/layers/82CF174C-FF88-41E4-ABFF-0316D2D30346?mode=design).

**We still need to set two things up:**
1. Connect the name with the username that's filled out at the top in the Pill container once the image is uploaded
2. We also need to add if they are VIP or not (pretty sure we don't have this set up, though?) 

If we can't set the two above items up, I can fix the design accordingly so it looks good.

I'm also only going off of how the mocks look in Abstract, because the Dropbox video they sent us doesn't show the animations for how the profile picture is added. I say we make a new ticket for the Animations.

Here's a gif of how it's looking now:
demo-large-4

File: src/Accounts/account.js [hunk 1]
```diff
@@ -19,15 +20,31 @@ export default class AccountDetails extends Component {
...
+
+  async prepareUserChanges() {
+    const {user, newProfilePic} = this.state
+    const changes = {...user}
+
+    if (newProfilePic) {
+      changes.profile_pic_url = await uploadImageToCloudinary(newProfilePic)
+    }
+
+    console.log('changes', changes)
```
Comment (daybreaker) [inline src/Accounts/account.js]: remove console.log

File: src/Accounts/account.js [hunk 2]
```diff
@@ -63,22 +92,19 @@ export default class AccountDetails extends Component {
...
         <View style={styles.paddingVerticalMedium}>
-
-          {false &&  // TODO: Re-enable when functionality is implemented.
           <View style={accountStyles.rowContainer}>
             <View style={accountStyles.row}>
               <View style={[ticketWalletStyles.avatarContainer, accountStyles.avatarContainer]}>
                 <Image
                   style={ticketWalletStyles.avatar}
-                  source={avatarPlaceholder}
+                  source={this.profilePicSourceToDisplay}
```
Comment (daybreaker) [inline src/Accounts/account.js]: does this show a placeholder if profilePicSourceToDisplay returns null?

File: src/constants/config.js [hunk 3]
```diff
@@ -2,6 +2,8 @@ import {Constants} from 'expo'
 
 const RELEASE_CHANNEL = Constants.manifest.releaseChannel || 'dev'
 const WEB_STAGING = 'https://beta.bigneon.com'
+const CLOUDINARY_CLOUD_STAGING = 'bigneon-dev'
+const CLOUDINARY_UPLOAD_PRESET_STAGING = 'dthcf8uc'
```
Comment (daybreaker) [inline src/constants/config.js]: do we have a different one for production? And does this get carried over into the production config if not?

File: src/state/authStateProvider.js [hunk 4]
```diff
@@ -3,6 +3,13 @@ import {AsyncStorage} from 'react-native'
 import {server, refreshWithToken, apiErrorAlert} from '../constants/Server'
 
 /* eslint-disable camelcase,space-before-function-paren */
+
+function shouldDoAdditionalSignUpStep(currentUser) {
+  console.log('currentUser', currentUser)
```
Comment (daybreaker) [inline src/state/authStateProvider.js]: another console

File: src/constants/config.js (hunk 3 above)
Comment (jwietelmann) [reply src/constants/config.js]: It gets carried over into the production config if not. I don't know if we have a different one for production or not. I just grabbed what I found when i inspected the request on beta web.

**Pull request merged**